README.md

```diff
@@ -224,6 +224,31 @@ assert("a" in #{ a: 1, b: 2 });
 
 See the [appendix](./NS-Proto-Appendix.md) to learn more about the `Record` & `Tuple` namespaces.
 
+## `Record` and `Tuple` Wrapper Objects
+
+Most users will not have to think about `Record` and `Tuple` wrapper objects. They exist for consistency with other ECMAScript features.
+
+- `Object(record)` creates an instance of `Record`, which is the wrapper object for `record` values. These instances are not extensible.
+- `Object(tuple)` creates an instance of `Tuple`, which is the wrapper object for `tuple` values. These instances are not extensible.
+
+`Record.prototype` is an ordinary object, is not a `Record` instance, and its prototype is `Object.prototype`. `record` values are not an instances of the `Record` prototype.
+
+`Tuple.prototype` is an ordinary object, is not a `Tuple` instance, and its prototype is `Object.prototype`. `tuple` values are not instances of the `Tuple` prototype.
+
+Accessing a member expression of a tuple or record via `.` or `[]` follows the standard [`GetValue`](https://tc39.es/ecma262/#sec-getvalue) semantics, and implicitly converts to an instance of the corresponding wrapper type.
+
+An instance of `Record` has the same keys and values as the `record` value it was created from. These keys are all `writable: false, enumerable: true, configurable: false`.
+
```

Comment:
In addition, there is a `length` key.
->
In addition, there is a non-enumerable `length` key.